testsuite/MDAnalysisTests/test_atomgroup.py

```diff
@@ -1724,7 +1724,18 @@ def test_set_dimensions(self):
         u.dimensions = np.array([10, 11, 12, 90, 90, 90])
         assert_allclose(u.dimensions, box)
 
+    def test_universe_kwargs(self):
+        u = MDAnalysis.Universe(PSF, PDB_small, fake_kwarg=True)
+        assert_equal(len(u.atoms), 3341, "Loading universe failed somehow")
+
+        assert_(u._kwargs['fake_kwarg'] is True)
 
+        # initialize new universe from pieces of existing one
+        u2 = MDAnalysis.Universe(u.filename, u.trajectory.filename, 
+                                 **u._kwargs)
+        
+        assert_(u2._kwargs['fake_kwarg'] is True)
```

Comment:
That looks good to me. Would it be to hard to compare the complete dictionaries or `u` and `u2`.